free_fleet_adapter/free_fleet_adapter/nav2_robot_adapter.py

```diff
@@ -220,16 +222,17 @@ def navigate(self, destination, execution):
             behavior_tree=""
         )
 
+        print("about to get the action")
```

Comment:
We should probably remove testing printouts, unless you find it necessary, we could convert it into a `self.node.get_logger().debug(...`, with more information about the goal message